cheat/cheatsheets/git

```diff
@@ -17,9 +17,16 @@ git commit -m "Your commit message"
 # To edit previous commit message
 git commit --amend
 
+# Git commit in the past
+git commit --date="`date --date='2 day ago'`"
```

Comment:
Jessie comes with git 2.1.4 and approxidate for `git commit` already works on version 2.1.0 (downgraded git package on Arch).